youtokentome/youtokentome.py

```diff
@@ -76,5 +76,7 @@ def subword_to_id(self, subword: str) -> int:
     def id_to_subword(self, id: int) -> str:
         return self.bpe_cython.id_to_subword(id)
 
-    def decode(self, ids: List[int], ignore_ids: Optional[List[int]] = None) -> str:
+    def decode(
+        self, ids: List[int], ignore_ids: Optional[Union[Set[int], List[int]]] = None
```

Comment:
Why is this union of `List` and `Set`? I think, it is better to check for `collections.abc.Iterable`.

Reply:
Dixed